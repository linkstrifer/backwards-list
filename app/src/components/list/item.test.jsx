import React from 'react';
import renderer from 'react-test-renderer';

import ItemComponent from './item.component';

test('Item snapshot', () => {
  const component = renderer.create(<ItemComponent label="test item" />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
