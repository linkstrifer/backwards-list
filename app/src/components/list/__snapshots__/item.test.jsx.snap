// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item snapshot 1`] = `
<li>
  test item
</li>
`;
